panther_docking/launch/docking.launch.py

```diff
@@ -102,29 +119,40 @@ def generate_launch_description():
         condition=IfCondition(use_docking),
     )
 
-    # FIXME: This launch does not work with the simulation. It can be caused by different versions of opencv
-    # station_launch = IncludeLaunchDescription(
-    #     PythonLaunchDescriptionSource(
-    #         PathJoinSubstitution(
-    #             [
-    #                 FindPackageShare("panther_docking"),
-    #                 "launch",
-    #                 "station.launch.py",
-    #             ]
-    #         ),
-    #     ),
-    #     launch_arguments={"namespace": namespace}.items(),
-    #     condition=UnlessCondition(use_sim),
-    # )
+    station_launch = IncludeLaunchDescription(
```

Comment:
For simulation, station should be spawn in `simulation.launch.py` instead of `spawn_robot.launch.py`. Potential isue when spawning two robots.

Reply:
Please do not consider simulation here. I fixed the simulation with tests with nav2 what will be pushed in the next PR ;)

Reply:
FYI: In lynx branch we decide to spawn robot separately by spawn_robot.launch.py, because there is quite difficult for example to specified multiple robots (panther/lynx) and each can have different equipment.